contracts/stake/src/contract.rs

```diff
@@ -70,6 +72,13 @@ pub trait StakingTrait {
 
     fn query_withdrawable_rewards(env: Env, address: Address) -> WithdrawableRewardsResponse;
 
+    fn query_withdrawable_rewards_ch(
```

Comment:
why are you using shortened `query_withdrawable_rewards_ch` instead of `query_withdrawable_rewards_chunks`